pyccl/halos/profiles/profile_base.py

```diff
@@ -47,35 +47,31 @@ def __init__(self):
                             "_real or _fourier implementation.")
         self.precision_fftlog = FFTLogParams()
 
-    @property
-    @abstractmethod
-    def normprof(self) -> bool:
-        """Normalize the profile in auto- and cross-correlations by
-        :math:`I^0_1(k\\rightarrow 0, a|u)`
-        (see :meth:`~pyccl.halos.halo_model.HMCalculator.I_0_1`).
-        """
-
-    # TODO: CCLv3 - Rename & allocate _normprof_bool to the subclasses.
+    def get_normalization(self, cosmo, a, hmc):
```

Comment:
This again departs from typical CCL arg ordering, so I think the best course of action is to make everything kw-only. I don't like making these simple things kw-only, that's why in the original version I had encapsulated a function within a function - the outer one accepted `hmc` and the inner one `cosmo` and `a`.

Reply:
I'll make hmc kwonly and default to None. They shouldn't always be required.

Reply:
Just note that in the (simplest) default case that we return 1, none of them is required.

Reply:
This is done now